core/ecschema-editing/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
   "compilerOptions": {
-    "outDir": "./lib"
+    "outDir": "./lib/cjs",
```

Comment:
Why are we doing this here but in none of the others?

Reply:
reverting; added here for some pkgs that deliver only cjs in order to not add a new build:cjs script, whch we migth want to remove later on...

there were earlier discussion of running maybe build:cjs in ci's if it was too slow to build both